src/components/TicketFeedContainer/TicketFeedContainer.tsx

```diff
@@ -1,26 +1,49 @@
-import { Box } from "@mui/material"
-import { InsightDataInterface } from "../../../interfaces/InsightData"
+import {Box, Button} from "@mui/material"
 import InsightCard from "../InsightCard/InsightCard"
+import {Insight} from "@interfaces/insight";
+import {FiledRequest} from "@interfaces/filedRequest";
+import {useState} from "react";
+import FiledRequestCard from "@components/FiledRequestCard/FiledRequestCard";
 
-interface TicketFeedContainerInterface { 
-    insightData: InsightDataInterface[]
+interface TicketFeedContainerInterface {
+    filedRequestData: {[id: string]: FiledRequest},
+    deleteFiledRequestHandlerGenerator: Function,
```

Comment:
What does generator mean here? Also please add return types

Reply:
Function that generates a function (that handles filed request deletes). Added explicit signature which hopefully clears it up